packages/components-css/icon-css/src/_mixin.scss

```diff
@@ -0,0 +1,21 @@
+@mixin nl-icon {
+  block-size: var(--nl-icon-block-size, 1em);
+  color: var(--nl-icon-color, currentcolor);
+  display: inline-block;
+  font-size: var(--nl-icon-font-size, 1em);
+  inline-size: var(--nl-icon-inline-size, 1em);
+  inset-block-start: var(--nl-icon-inset-block-start, 0);
```

Comment:
Waarom is enkel de `-start` configureerbaar? waarom niet ook `-end`? (en de `inline` varianten?)

Reply:
Ja, deze had je mogen negeren. Met de CSS is nog niet echt wat gedaan. Dit is nagenoeg copy-pasta van de urecht component.